src/client/pages/Bodem/Bodem.test.tsx

```diff
@@ -126,37 +168,109 @@ const testState: any = {
               status: 'Afgehandeld',
               id: '',
               datePublished: '',
-              isActive: false,
-              isChecked: false,
+              isActive: true,
+              isChecked: true,
             },
           ],
         },
       ],
     },
     status: 'OK',
   },
-};
+} as unknown as AppState;
+
+const createComponent = componentCreator({
+  component: Bodem,
+  routeEntry: AppRoutes.BODEM,
+  routePath: AppRoutes.BODEM,
+});
 
 describe('Bodem', () => {
-  describe('with results', () => {
-    const renderBodem = setupMockApp(Bodem, 'BODEM', testState);
+  describe('With aanvragen', () => {
+    const MockBodem = createComponent(testState);
+
+    describe('Tables', () => {
+      const screen = render(<MockBodem />);
+
+      const lopendeAanvraagTableHeader = screen.getByRole('heading', {
+        name: 'Lopende aanvragen',
+      });
+
+      const afgehandeldeAanvraagTableHeader = screen.getByRole('heading', {
+        name: 'Afgehandelde aanvragen',
+      });
+
+      test('Headers are found', () => {
+        expect(lopendeAanvraagTableHeader).toBeInTheDocument();
+        expect(afgehandeldeAanvraagTableHeader).toBeInTheDocument();
+      });
+
+      const lopendePattern = /(In behandeling)|Ontvangen/;
+      const afgehandeldePattern = /Afgehandeld|Afgewezen/;
+
+      const lopendeAanvraagTable =
+        lopendeAanvraagTableHeader.nextSibling as HTMLElement;
+      if (!lopendeAanvraagTable) {
+        throw Error('No sibling found');
+      }
+
+      const afgehandeldeAanvraagTable =
+        afgehandeldeAanvraagTableHeader.nextSibling as HTMLElement;
+      if (!afgehandeldeAanvraagTable) {
+        throw Error('No sibling found');
+      }
 
-    it('should show the right overviewpage', () => {
-      const { asFragment } = renderBodem();
+      test('Table items found', () => {
+        const lopendeAanvraagRows = within(lopendeAanvraagTable).getAllByRole(
+          'row',
+          {
+            name: lopendePattern,
+          }
+        );
+        expect(lopendeAanvraagRows.length).toBe(2);
+
+        const afgehandeldeAanvraagRows = within(
+          afgehandeldeAanvraagTable
+        ).queryAllByRole('row', {
+          name: afgehandeldePattern,
+        });
+        expect(afgehandeldeAanvraagRows.length).toBe(2);
```

Comment:
(meerdere keren)
Beter om de build in functie te gebruiken voor het geval afgehandeldeAanvraagRows `null `is:
```suggestion
        expect(afgehandeldeAanvraagRows).toHaveLength(2);
```